doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -481,6 +481,12 @@ Enhanced ShockBurst (ESB)
 
 The issues in this section are related to the :ref:`ug_esb` protocol.
 
+.. rst-class:: v2.9
+
+NCSDK-30802: Packet retransmission is not working properly in the ESB for the nRF54 series.
+
```

Comment:
```suggestion
NCSDK-30802: Packet retransmission is not working properly in the ESB for the nRF54H Series devices
  An nRF54H device suddenly stops transmitting ESB packets.

```
- The first line with the Jira issue never ends with a fullstop.
- Another line (indented two spaces) is needed to give a bit more information. Please improve it, if this proposal not good enough. It comes from the Jira issue.